tests/unit_tests/test_filters.py

```diff
@@ -294,3 +294,43 @@ def test_tabular_from_energyfilter():
 
     tab = efilter.get_tabular(values=np.array([10, 10, 5]), interpolation='linear-linear')
     assert tab.interpolation == 'linear-linear'
+
+
+def test_SurfaceFilter_CompositeSurface(run_in_tmpdir, box_model):
+    m = openmc.Material()
+    m.add_nuclide('U235', 1.0)
+    m.set_density('g/cm3', 1.0)
+
+    box = openmc.model.RectangularPrism(10., 10., boundary_type='vacuum')
+    c = openmc.Cell(fill=m, region=-box)
+    box_model.geometry.root_universe = openmc.Universe(cells=[c])
+    
+    tally = openmc.Tally()
+    tally.filters = [openmc.SurfaceFilter(box.get_surfaces())]
```

Comment:
I think better yet we'd be able to pass a `CompositeSurface` directly to the `SurfaceFilter` class, which might require some additional handling in that object.

Reply:
Yes, I think it is clearer. It is already done. I thought it was easier to change the `SurfaceFilter` class than to make a `CompositeSurface` always return a list of surfaces. Let me know what you think.